client/protocols/autostyle.py

```diff
@@ -15,6 +15,7 @@ class AutoStyleProtocol(models.Protocol):
 
     # Timeouts are specified in seconds.
     SHORT_TIMEOUT = 10
+    RETRY_LIMIT = 5
```

Comment:
Nope... I thought it was like a thing that somebody accessing the class would use.